### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by CikiMomogi**

### CODE REVIEW

File: .github/workflows/translation-guard.yml [hunk 1]
```diff
@@ -27,10 +27,14 @@ jobs:
           if has_label "ready for translation"; then
               for language_label in $language_labels; do
                   if ! has_label "$language_label"; then
+                      echo "Error: Missing language label '$language_label'"
                       exit 1
+                  else
+                      echo "Found language label '$language_label'"
```
Comment (roj1512) [inline .github/workflows/translation-guard.yml:33]: This is not necessary.

**Comment by KnorpelSenf** [comment]
no idea

**Review by roj1512 — COMMENTED** [review]
Can you just undo all changes other than fixing the typo?

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you so much!

**Review by roj1512 — COMMENTED** [review]
Can you also change `name: test` to `name: Translation Guard`?

**Pull request merged**